bindings/jupyter-modules/modules/jupyter-graph-builder/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@mlvis/jupyter-graph-builder",
+  "version": "0.0.1",
```

Comment:
nit: should start with 1.0.0 but will be synced once we use lerna to bump the versions.